src/states/render-messages.ts

```diff
@@ -198,6 +198,35 @@ export const renderAmizoneMenu = () => ({
   },
 });
 
+export const renderReplyMessage = () => ({
```

Comment:
This looks fine, but I can't see anywhere you're sending this payload to the WhatsApp API.

Reply:
Yes that was the underlying issue, to get it to send the payload.
Im trying to use the `HandleScheduleInput()` function directly and render the schedule for Today/Yesterdays Date. I have to somehow trigger the event after the button is clicked (which i'll have to figure out how).

Reply:
You can insert a call like so before rendering the menu
```js
await ctx.bot.sendInteractiveMessage(payload.sender, renderReplyMessage());
```
https://github.com/asetalias/amibot/blob/main/src/states/state-handlers.ts#L214